app/serializers/item_serializer.rb

```diff
@@ -1,3 +1,7 @@
 class ItemSerializer < ActiveModel::Serializer
   attributes :id, :name, :description, :unit_price, :merchant_id
+
+  def unit_price
+    object.unit_price.to_f / 100
+  end
 end
```

Comment:
@s-espinosa it's just math! Wondering if that's ok vs. using ruby to do fancy things or manipulating it as a string.